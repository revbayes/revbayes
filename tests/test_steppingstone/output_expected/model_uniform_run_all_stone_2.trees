Iteration	Posterior	Likelihood	Prior	psi
0	-21335.42	-21312.67	-22.75015	(Galeopterus_variegatus[&index=23]:0.353658,Tarsius_syrichta[&index=21]:0.413762,(((Daubentonia_madagascariensis[&index=7]:0.285568,(Propithecus_coquereli[&index=19]:0.140788,(((Cheirogaleus_major[&index=4]:0.075674,Microcebus_murinus[&index=14]:0.230988)[&index=35]:0.124814,Lepilemur_hubbardorum[&index=11]:0.227317)[&index=39]:0.047137,(Lemur_catta[&index=10]:0.194980,Varecia_variegata_variegata[&index=22]:0.213053)[&index=38]:0.052995)[&index=40]:0.024331)[&index=27]:0.187835)[&index=34]:0.006193,((Loris_tardigradus[&index=12]:0.264360,Nycticebus_coucang[&index=15]:0.234041)[&index=37]:0.067727,((Galago_senegalensis[&index=8]:0.127563,Otolemur_crassicaudatus[&index=16]:0.101916)[&index=24]:0.163933,Perodicticus_potto[&index=18]:0.200155)[&index=29]:0.042361)[&index=41]:0.146203)[&index=28]:0.089669,((((Saimiri_sciureus[&index=20]:0.191332,Cebus_albifrons[&index=3]:0.245341)[&index=26]:0.001387,Aotus_trivirgatus[&index=1]:0.111163)[&index=33]:0.064258,Callicebus_donacophilus[&index=2]:0.147093)[&index=43]:0.312681,((Hylobates_lar[&index=9]:0.195169,Pan_paniscus[&index=17]:0.162841)[&index=32]:0.036438,(Colobus_guereza[&index=6]:0.151589,(Chlorocebus_aethiops[&index=5]:0.153365,Macaca_mulatta[&index=13]:0.165248)[&index=25]:0.093853)[&index=42]:0.234021)[&index=30]:0.374370)[&index=31]:0.172139)[&index=36]:0.101304)[&index=44];
10	-21319.86	-21293.69	-26.16581	(Galeopterus_variegatus[&index=23]:0.432448,(Tarsius_syrichta[&index=21]:0.283950,(Daubentonia_madagascariensis[&index=7]:0.244156,(((Nycticebus_coucang[&index=15]:0.208341,Loris_tardigradus[&index=12]:0.205118)[&index=37]:0.045628,(Perodicticus_potto[&index=18]:0.236017,(Galago_senegalensis[&index=8]:0.165821,Otolemur_crassicaudatus[&index=16]:0.171919)[&index=24]:0.102953)[&index=29]:0.030677)[&index=41]:0.161312,(Lepilemur_hubbardorum[&index=11]:0.290433,((Microcebus_murinus[&index=14]:0.307399,Cheirogaleus_major[&index=4]:0.090970)[&index=35]:0.133130,((Varecia_variegata_variegata[&index=22]:0.262468,Lemur_catta[&index=10]:0.149221)[&index=38]:0.039075,Propithecus_coquereli[&index=19]:0.264819)[&index=40]:0.038220)[&index=39]:0.042777)[&index=27]:0.196784)[&index=34]:0.031185)[&index=28]:0.073992)[&index=36]:0.078906,(((Hylobates_lar[&index=9]:0.211298,Pan_paniscus[&index=17]:0.147663)[&index=32]:0.050634,((Macaca_mulatta[&index=13]:0.196894,Chlorocebus_aethiops[&index=5]:0.150327)[&index=25]:0.083046,Colobus_guereza[&index=6]:0.218018)[&index=42]:0.165371)[&index=30]:0.345519,((Aotus_trivirgatus[&index=1]:0.127348,(Cebus_albifrons[&index=3]:0.214521,Saimiri_sciureus[&index=20]:0.244087)[&index=26]:0.004627)[&index=33]:0.073633,Callicebus_donacophilus[&index=2]:0.206610)[&index=43]:0.203374)[&index=31]:0.265147)[&index=44];
