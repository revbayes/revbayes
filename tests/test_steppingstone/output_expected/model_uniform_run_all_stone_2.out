state	power	likelihood
4	0.554929	-21282.6
5	0.554929	-21282.6
6	0.554929	-21286
7	0.554929	-21293.6
8	0.554929	-21294.4
9	0.554929	-21295.4
10	0.554929	-21284.4
